prac_05/word_occurences.py

```diff
@@ -0,0 +1,17 @@
+word_count = {}
+input_text = input("Text: ")
+input_text = input_text.split(' ')
+word_list = []
+word_length = 0
+for word in input_text:
+    if word in word_count:
+        word_count[word] = word_count[word] + 1
+    else:
+        word_count[word] = 1
+        word_list.append(word)
+        if len(word) > word_length:
```

Comment:
You could find the length of the longest word by just using longest_word = len(max(word_list))